bool load
---
float32 backward_310L
float32 backward_310R
float32 backward_300L
float32 backward_300R
float32 backward_250L
float32 backward_250R
float32 backward_200L
float32 backward_200R
float32 backward_150L
float32 backward_150R	
float32 backward_100L
float32 backward_100R
float32 backward_50L
float32 backward_50R
int16 forward_x_add
int16 forward_x_max
int16 backward_x_add
int16 backward_x_max
int16 forward_initial_x
int16 backward_initial_x
int16 slowdown_250
int16 slowdown_300
int16 slowdown_initial_x
int16 FRAMCOUNT
int16 change_mode
int32 back_total_size
int16 forward_initial_theta
int16 forward_theta_left_one
int16 forward_theta_right_one
int16 forward_theta_left_two
int16 forward_theta_right_two
int16 forward_theta_left_three
int16 forward_theta_right_three
int16 backward_initial_theta
int16 backward_theta_left_one
int16 backward_theta_right_one
int16 backward_theta_left_two
int16 backward_theta_right_two
int16 backward_theta_left_three
int16 backward_theta_right_three
int16 delay_ybtime
int16 without_left
int16 without_right
int16 right_timeuse
int16 initial_timeuse
int16 left_timeuse
float32 IMU_angle_range
